File: js/node/factory/Factory.js
require('./pizzaStore/NYPizzaStore');
require('./pizzaStore/ChicagoPizzaStore');

var nyStore = new NYPizzaStore();
var chicagoStore = new ChicagoPizzaStore();

var pizza = nyStore.orderPizza("cheese");
console.log("Ethan ordered a " + pizza);

pizza = chicagoStore.orderPizza("cheese");
console.log("Joel ordered a " + pizza);

var pizza = nyStore.orderPizza("clam");
console.log("Ethan ordered a " + pizza);

pizza = chicagoStore.orderPizza("clam");
console.log("Joel ordered a " + pizza);

var pizza = nyStore.orderPizza("pepperoni");
console.log("Ethan ordered a " + pizza);

pizza = chicagoStore.orderPizza("pepperoni");
console.log("Joel ordered a " + pizza);

var pizza = nyStore.orderPizza("veggie");
console.log("Ethan ordered a " + pizza);

pizza = chicagoStore.orderPizza("veggie");
console.log("Joel ordered a " + pizza);

File: python/2.7/factory/Factory.py
import sys

sys.path.append("./pizzaStore")
from NYPizzaStore import *
from ChicagoPizzaStore import *

if __name__ == "__main__":
    nyStore = NYPizzaStore()
    chicagoStore = ChicagoPizzaStore()

    pizza = nyStore.orderPizza("cheese")
    print "Ethan ordered a %s\n" %(pizza)

    pizza = chicagoStore.orderPizza("cheese")
    print "Joel ordered a %s\n" %(pizza)

    pizza = nyStore.orderPizza("clam")
    print "Ethan ordered a %s\n" %(pizza)

    pizza = chicagoStore.orderPizza("clam")
    print "Joel ordered a %s\n" %(pizza)

    pizza = nyStore.orderPizza("pepperoni")
    print "Ethan ordered a %s\n" %(pizza)

    pizza = chicagoStore.orderPizza("pepperoni")
    print "Joel ordered a %s\n" %(pizza)

    pizza = nyStore.orderPizza("veggie")
    print "Ethan ordered a %s\n" %(pizza)

    pizza = chicagoStore.orderPizza("veggie")
    print "Joel ordered a %s\n" %(pizza)